spring-cloud-gcp-pubsub-stream-binder/src/main/java/org/springframework/cloud/gcp/stream/binder/pubsub/provisioning/PubSubChannelProvisioner.java

```diff
@@ -118,7 +120,11 @@ private Topic makeSureTopicExists(String topicName, boolean autoCreate) {
 		Topic topic = this.pubSubAdmin.getTopic(topicName);
 		if (topic == null) {
 			if (autoCreate) {
-				topic = this.pubSubAdmin.createTopic(topicName);
+				try {
+					topic = this.pubSubAdmin.createTopic(topicName);
+				} catch(AlreadyExistsException alreadyExistsException) {
+					// Ignore concurrent topic creation - we're good as long as topic was created and exists
```

Comment:
Could you log a warning here?

Reply:
Sure

Reply:
Why warning level if it's not a real issue?
I would go with DEBUG, or at most INFO.

Reply:
Changed level to INFO according to @elefeint comment